lib/internal/v8_prof_processor.js

```diff
@@ -16,29 +11,19 @@ var scriptFiles = [
   'v8/tools/SourceMap',
   'v8/tools/tickprocessor-driver'
 ];
-var tempScript = 'tick-processor-tmp-' + process.pid;
-var tempNm = 'mac-nm-' + process.pid;
+var script = '';
 
-process.on('exit', function() {
-  try { fs.unlinkSync(tempScript); } catch (e) {}
-  try { fs.unlinkSync(tempNm); } catch (e) {}
-});
-process.on('uncaughtException', function(err) {
-  try { fs.unlinkSync(tempScript); } catch (e) {}
-  try { fs.unlinkSync(tempNm); } catch (e) {}
-  throw err;
+scriptFiles.forEach(function(s) {
+  script += process.binding('natives')[s] + '\n';
 });
 
-scriptFiles.forEach(function(script) {
-  fs.appendFileSync(tempScript, process.binding('natives')[script]);
-});
-var tickArguments = [tempScript];
+var tickArguments = [];
 if (process.platform === 'darwin') {
-  fs.writeFileSync(tempNm, process.binding('natives')['v8/tools/mac-nm'],
-    { mode: 0o555 });
-  tickArguments.push('--mac', '--nm=' + path.join(process.cwd(), tempNm));
+  var nm = 'foo() { nm "$@" | (c++filt -p -i || cat) }; foo';
```

Comment:
Hm, if we are not in strict mode, shouldn't we use var instead. Sloppy mode const is totally different IIRC